cnxepub/tests/test_collation.py

```diff
@@ -280,4 +280,4 @@ def test_with_ruleset(self):
         # Check for the appended composite document
         self.assertEqual(len(collated_binder), 3)
         self.assertEqual(collated_binder[2].metadata['title'],
-                         'Composite One')
+                         'Document One')
```

Comment:
As I wrote in the extended commit message, 
> Since the single-page is not a cannonical source and does not need to retain the complete information of the document, the title overrides in this case, ignoring the TOC during reconstitution is okay.

We don't have the ability to do this kind of creating and movement of TOC items in the ruleset just yet. Plus, during this collation process, the title of the document will always equal the TOC item title, because the title-override is used during rendering.